content/en/docs/instrumentation/erlang/sampling.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Sampling
+weight: 37
+---
+
+## Sampling
+
+[Sampling](/docs/concepts/sampling/) is a process that restricts the amount of
+traces that are generated by a system. The Erlang SDK offers several
+[head samplers](/docs/concepts/sampling#head-sampling).
+
+### Default behavior
+
+By default, all spans are sampled, and thus, 100% of traces are sampled. If you
+do not need to manage data volume, don't bother setting a sampler.
+
+### ParentBasedSampler
+
+When sampling, the `ParentBasedSampler` is most often used for
+[head sampler](/docs/concepts/sampling/#head-sampling). It uses the sampling
+decision of the Span's parent, or the fact that there is no parent, to know
+which secondary sampler to use.
+
+The sampler can be configured with environment variables `OTEL_TRACES_SAMPLER`
+and `OTEL_TRACES_SAMPLER_ARG` or using the Application configuration allows you
+to configure each of the 5 potential states of a Span's parent:
+
+- `root` - No parent
+- `remote_parent_sampled` - Parent is from a remote Span that is sampled
+- `remote_parent_not_sampled` - Parent is from a remote Span that is not sampled
+- `local_parent_sampled` - Parent is from a local Span that is sampled
+- `local_parent_not_sampled` - Parent is from a local Span that is not sampled
+
+#### TraceIdRatioBasedSampler
+
+Within the `ParentBasedSampler` the most common is the
+`TraceIdRatioBasedSampler`. It deterministically samples a percentage of traces
+that you pass in as a parameter.
+
+##### Environment Variables
+
+You can configure the `TraceIdRatioBasedSampler` with environment variables:
+
+```shell
+export OTEL_TRACES_SAMPLER="parentbased_traceidratio"
+export OTEL_TRACES_SAMPLER_ARG="0.1"
+```
```

Comment:
Earlier you define:

> - `root` - No parent
> - `remote_parent_sampled` - Parent is from a remote Span that is sampled
> - `remote_parent_not_sampled` - Parent is from a remote Span that is not sampled
> - `local_parent_sampled` - Parent is from a local Span that is sampled
> - `local_parent_not_sampled` - Parent is from a local Span that is not sampled

I see that these are defined lower for Erlang terms. Is there any way to configure these with the ENV vars method?

Reply:
There isn't. I see one place I can make this clearer, I'll update the paragraph in `ParentBasedSampler` section to be:

> The sampler can be configured with the environment variables `OTEL_TRACES_SAMPLER`
and the sampler to use when there is no parent in `OTEL_TRACES_SAMPLER_ARG`, or using the Application configuration allows you
to configure each of the 5 potential states of a Span's parent: